### ISSUE TITLE
Expose broadcastTx options

### ISSUE THREAD

**Issue by bartmacbartek** [cosmos/cosmjs#794]
Make `timeoutMs` and `pollIntervalMs` arguments of `this.broadcastTx(txBytes, timeoutMs, pollIntervalMs)` configurable from within `signAndBroadcast` method.

Currently, it's not possible:
https://github.com/cosmos/cosmjs/blob/7da03ad80f5434d2fdffe7d2f8f29415a61da567/packages/cosmwasm-stargate/src/signingcosmwasmclient.ts#L425-L434

**Comment by willclarktech** [comment]
@webmaster128 Should we just add it to the options when instantiating the client?

**Comment by webmaster128** [comment]
Yeah I was thinking something like that. If we do that, do we even need the arguments in `broadcastTx`? If we keep them in there, we only need the option in the *SigningClient not in the read-only client.

**Comment by webmaster128** [comment]
It is pulic in the read-only clients. So I'd keep the argument there as-is. Then we just need to change all usages of it in `Signing*Client`
